Task-based edit: Facebook Audience Export (#6204)
braze-inc/braze-docs: _docs/_partners/message_orchestration/additional_channels/retargeting/facebook.md

# Facebook Audience export

> The Braze and Facebook integration allows you to manually export your Braze segments to Facebook to create Facebook Custom Audiences. This is a one-time, static audience export and will only create new Facebook Custom Audiences.

Common use cases for exporting Facebook Custom Audiences include:
- Retargetting users at specific points within their lifecycle
- Creating exclusion targeting lists
- Creating [Lookalike Audiences][4] to acquire new users more efficiently
<br><br>

{% alert note %}
The Facebook audience export uses the **User Access Token** to authorize requests.<br><br>

> If you are using this feature alongside the [Audience Sync to Facebook]({{site.baseurl}}/audience_sync_facebook/) feature (currently in beta), Braze will default to using the more reliable **System User Token** that you have already generated, to authorize requests.

Reviewer: ```suggestion
If you are using this feature alongside the [Audience Sync to Facebook]({{site.baseurl}}/audience_sync_facebook/) feature (currently in beta), Braze will default to using the more reliable **System User Token** that you have already generated, to authorize requests.
```
Audience Sync to Facebook is generally available now (there's no beta callout in the article)

{% alert important %}
For customers that have previously undergone the Facebook app review process for [Ads Management](https://developers.facebook.com/docs/facebook-login/permissions/#reference-ads_management) and [Ads Management Standard Access](https://developers.facebook.com/docs/marketing-api/access#standard), your system user token will still be valid for the Facebook audience step. You will not be able to edit or revoke the Facebook system user token through the Facebook partner page. Instead, you can connect your Facebook account to replace your Facebook system user token within your Braze workspace. 

<br><br>The new Facebook oAuth configuration will also apply to [Facebook exports via segments]({{site.baseurl}}/partners/message_orchestration/additional_channels/retargeting/facebook/#prerequisites). 
{% endalert %}

### Step 2: Export your users into Facebook

> Braze's Facebook audience export is accessible through the **Segments** page. 

Reviewer: ```suggestion
In Braze, Facebook audience export is accessible through the **Segments** page. 
```

1. On the **Segments** page, click the gear next to the segment you'd like to export.
2. Click **Export as Facebook Audience**. <br><br>![A list of Braze segments. For the first segment, the setting symbol is selected and the "Export as Facebook Audience" button is shown.][11]

{: start="3"}

> 3. If you haven't already activated Facebook within Braze, you will be prompted to go to the Facebook Technology Partners page in the dashboard. If you already activated Facebook through **Technology Partners > Facebook**, you will be able to select the user field to export, and a dropdown to select your Facebook ad account will show. <br><br> There are three possible user fields you can export:  

Reviewer: ```suggestion
3. If you haven't already activated Facebook within Braze, you will be prompted to go to the Facebook Technology Partners page in the dashboard. If you already activated Facebook through **Technology Partners** > **Facebook**, you will be able to select the user field to export, and a dropdown to select your Facebook ad account will show. <br><br> There are three possible user fields you can export:  
```

- The size of the Custom Audience. [Learn more.](https://marketingland.com/exclusive-facebook-will-no-longer-show-audience-reach-estimates-for-custom-audiences-after-vulnerability-detected-236923)
{% endalert %}

#### Configuring your audience export

When building Facebook audiences, you may wish to include or exclude certain users based on their preferences, and in order to comply with privacy laws, such as the “Do Not Sell or Share” right under the [CCPA](https://oag.ca.gov/privacy/ccpa). Marketers should implement the relevant filters for users’ eligibility within their Canvas entry criteria. Below we list some options. 

- If you have collected the [iOS IDFA through the Braze SDK]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/initial_sdk_setup/other_sdk_customizations/#optional-idfa-collection), you will be able to use the **Ads Tracking Enabled** filter. Select the value as `true` to only send users into Audience Sync destinations where they have opted in. 

![][16]{: style="max-width:75%;"}

> - If you are collecting ‘opt-ins’,  ‘opt-outs’, ‘Do Not Sell Or Share’ or any other relevant custom attributes, you should include these within your Canvas entry criteria as a filter: 

Reviewer: ```suggestion
- If you are collecting ‘opt-ins’, ‘opt-outs’, ‘Do Not Sell Or Share’, or any other relevant custom attributes, you should include these within your Canvas entry criteria as a filter:
```
Might want to check the formatting for: ‘opt-ins’, ‘opt-outs’, ‘Do Not Sell Or Share’,
